lxc/export.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"io"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	lxd "github.com/lxc/lxd/client"
+	"github.com/lxc/lxd/lxc/utils"
+	"github.com/lxc/lxd/shared/api"
+	cli "github.com/lxc/lxd/shared/cmd"
+	"github.com/lxc/lxd/shared/i18n"
+)
+
+type cmdExport struct {
+	global *cmdGlobal
+
+	flagContainerOnly    bool
+	flagOptimizedStorage bool
+}
+
+func (c *cmdExport) Command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = i18n.G("export [<remote>:]<container> [target] [--container-only] [--optimized-storage]")
+	cmd.Short = i18n.G("Export container backup")
+	cmd.Long = cli.FormatSection(i18n.G("Description"), i18n.G(
+		`Export backups of containers including their snapshots.`))
+	cmd.Example = cli.FormatSection("", i18n.G(
+		`lxc export u1 backup0.tar.xz
+    Create the backup.
+
+lxc import backup0
+    Export the backup.`))
+
+	cmd.RunE = c.Run
+	cmd.Flags().BoolVar(&c.flagContainerOnly, "container-only", false,
+		i18n.G("Whether or not to only backup the container (without snapshots)"))
+	cmd.Flags().BoolVar(&c.flagOptimizedStorage, "optimized-storage", false,
+		i18n.G("Use send command for containers and backups (only Btrfs and ZFS)"))
```

Comment:
That's much too specific, we should say something like `Use storage driver specific optimized format (can only be restored on a similar pool)`

Reply:
fixed